app/src/main/java/com/support/android/designlibdemo/CheeseDetailActivity.java

```diff
@@ -42,6 +43,17 @@ public void onCreate(Bundle savedInstanceState) {
         final String cheeseName = intent.getStringExtra(EXTRA_NAME);
 
         final Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        if(Build.VERSION.SDK_INT>=21){
```

Comment:
Change to `Build.VERSION.SDK_INT == 21` as this issue is only present at that API level.